### PR TITLE
Fixes #72: SI-SDR sign in eval

### LINKED ISSUE #72: Docs on Model Evaluation pipeline
I am trying to reproduce the DAC training and evaluation. 
I am getting negative values for SI-SDR loss even for the pretrained weights provided in the repo.
Could you please provide documentation for evaluating the model?
Thanks

### PR DESCRIPTION
**Pull request by ivnvalex**
Fixes #72: SI-SDR sign in eval
`evaluate.py` uses `SISDRLoss`, which is `sdr = -10 * torch.log10(signal / noise + eps)` inside, negative SDR, obviously, for a loss function objective, but in evaluation we just need negative that.

### CODE REVIEW

**Comment by ivnvalex** [comment]
@eeishaan @pseeth @ritheshkumar95
Thanks for contributing DAC to the community!
Here is a little fix, ready for review c:
